docs/build.gradle

```diff
@@ -57,3 +53,8 @@ asciidoctor {
     }
   }
 }
+
+liveReload {
+  docRoot asciidoctor.outputDir.canonicalPath
```

Comment:
Why `.canonicalPath`?

Reply:
Same answer, it's in the example

Reply:
Fair enough - it looks like they're using strings for file paths rather than file (or path) objects.